guides/common/assembly_using-external-databases.adoc

```diff
@@ -14,8 +14,14 @@ endif::[]
 
 To create and use external databases for {Project}, you must complete the following procedures:
 
+ifdef::orcharhino[]
 . xref:preparing-a-host-for-external-databases_{context}[].
-Prepare a {RHEL} 7 server to host the external databases.
+Prepare a {EL} 7 server to host the external databases.
+endif::[]
+ifndef::orcharhino[]
+. xref:preparing-a-host-for-external-databases_{context}[].
```

Comment:
This part is the same for both `ifdef` and `ifndef`. Can't this be pulled out?

Reply:
Addressed :+1: Please review.